app/src/sharedTest/java/org/oppia/app/testing/NavigationDrawerTestActivityTest.kt

```diff
@@ -137,6 +141,34 @@ class NavigationDrawerTestActivityTest {
     }
   }
 
+  @Test
+  fun testNavigationDrawerTestActivity_withAdminProfile_openNavigationDrawer_checkAdministratorControlsDisplayed() {
+    launch<NavigationDrawerTestActivity>(createNavigationDrawerActivityIntent(0)).use {
+      onView(withContentDescription(R.string.drawer_open_content_description)).perform(click())
+      onView(withId(R.id.administrator_controls_linear_layout)).check(matches(isDisplayed()))
+    }
+  }
+
+  @Test
+  fun testNavigationDrawerTestActivity_withAdminProfile_openNavigationDrawer_clickAdministratorControls_checkOpensAdministratorControlsActivity_onBackPressed_showsHomeActivity() {
```

Comment:
Actually this test case is correct but this test case is not a functionality of NavigationDrawer, instead it is the functionality of AdministratorControlsActivity. So please create a new dedicated test file `AdministratorControlsActivityTest` and in this file write this test case. Also, note that the function name will be updated accordingly.

Reply:
I have made anew test file and written same testcase in it. Test is passing successfully.